assets/js/app/DataLayer.js

```diff
@@ -112,12 +112,13 @@ LocusZoom.PositionsDataLayer = function(){
 
     LocusZoom.DataLayer.apply(this, arguments);  
     this.id = "positions";
-    this.fields = ["id","position","pvalue","refAllele","ld:state"];
+    this.xfield = "position";
+    this.yfield = "pvalue|neglog10";
+    this.fields = ["id", this.xfield, this.yfield, "refAllele", "ld:state"];
 
     this.postget = function(){
         this.data.map(function(d, i){
             this.data[i].ld = +d["ld:state"];
```

Comment:
The unary plus operator for number conversion is, presently, the "fastest and preferred way of converting something into a number" (according to [developer.mozilla.org](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Unary_plus)). So yes, essentially all this does is ensure the value is a number type, and is shorter syntax and faster execution than `Number()`. It's also more type-safe than `parseInt` or `parseFloat` as certain values like the empty string ("") will return `NaN` with those parse functions but the unary plus will return `0`.

I wouldn't think a `|numeric` pipe would be necessary since a single character unary operator will always be much faster than parsing out and executing a transformation function.